### ISSUE TITLE
Instance type recommendations 

### ISSUE THREAD

**Issue by robertmassaioli** [m-chandler/factorio-spot-pricing#32]
I was reviewing instance type pricing and here are my conclusions:

[ ] m5zn.large - 4.5ghz clock speed and only 7-8c per hour spot (best possible performance)
[ ] m6a.large - 3.6ghz clock speed and 8gb while under 5c spot (this is practically the same price as the next two and much better clock speed)
[ ] m7g.large - latest generation, compute optimised. Unknown clock speed, 2.6ghz clock speed (slow), 3c spot 8GB ram!! (Cloud halve the ram with a c7g.large but what is the point when the m7g has a slightly cheaper 0.1c spot prices)  (this looks like a better modern default than m3.medium)
[ ] m3.medium - 2.5ghz, slow, 3c spot

In short, of I updated the readme with some instance recommendations, would you be against that? Also, would you consider making m7g.large or m6a.large the new default? I would recommend m6a.large as the new default though I plan on testing it over the next few days.

**Comment by robertmassaioli** [comment]
![Screenshot 2024-10-09 at 1 29 44 pm](https://github.com/user-attachments/assets/7c3826f8-ed1c-4e12-ab56-97ea9874e514)

Yep, m6a.large worked great and was just as cheap. I would recommend that instead.

**Comment by m-chandler** [comment]
Hey @robertmassaioli, I'd certainly be open to changing the default to something more modern. I wouldn't use the m3.medium these days.

BUT, I don't know if you've seen the latest commits. I've changed the template so you no longer need to specify an instance type for spot. Just specify the desired vCPU / Memory / Max Price and it will find the best instance. Of course you're still given the option to force a specific instance if you wish.

Sorry, it would probably be more useful if I included a changelog section in the readme and include this detail. I'll do this later.

Happy to receive a PR, as the m3.medium is still mentioned. A few areas of the readme probably need to be re-written. I did rush that last commit and doco is lacking.

**Comment by robertmassaioli** [comment]
Ok, I think that I will do that then. I think that CPU speed is important so I'll probably still select the instance type manually. And I'll add recommendations.

**Comment by m-chandler** [comment]
To be fair I probably didn't get far enough in the game / have enough mods / have enough friends🥲for the clock speed to become an issue.

If you'd like to do a PR adding some recommendations that would be good. Keeping in mind that the spot price will differ across regions, but an example price of Sydney should be fine.

**Comment by robertmassaioli** [comment]
> If you'd like to do a PR adding some recommendations that would be good. Keeping in mind that the spot price will differ across regions, but an example price of Sydney should be fine.

Okay, I'll do that! Stand by. :)

**Comment by m-chandler** [comment]
Resolved by https://github.com/m-chandler/factorio-spot-pricing/commit/c7d5dcb1b3adae14cecf3dbf2bcdd11e1480aecf
